station.go

```diff
@@ -614,3 +631,49 @@ func (sd *schemaDetails) validateGraphQlMsg(msg any) ([]byte, error) {
 	}
 	return msgBytes, nil
 }
+
+func (sd *schemaDetails) validAvroSchemaMsg(msg any) ([]byte, error) {	
+	var (
+		msgBytes []byte
+		err      error
+		message  interface{}
+	)
+
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	switch msg.(type) {
+	case []byte:
+		msgBytes = msg.([]byte)
+		if err := json.Unmarshal(msgBytes, &message); err != nil {
```

Comment:
myData := []byte(`{"username": "john", "age": 30}`)
When json unmarshal the above data, `int` type is converted to `float64` thus causing schema validation error. So when user is defining avro schema, schema can't have int type. It should be double rather than int.

https://github.com/hamba/avro#types-conversions

<img width="656" alt="Screenshot 2023-07-26 at 1 18 08 PM" src="https://github.com/memphisdev/memphis.go/assets/30820063/75d5df46-0c32-4ef6-adc7-3b1e875c6f78">

Reply:
Any workaround to solve this? otherwise we should mention it on the UI as well

Reply:
> Any workaround to solve this? otherwise we should mention it on the UI as well

I can think of the below solution. But this solution converts int to int64. Still we need to add note in the UI.
https://go.dev/play/p/SkG6qlolz1B
```
myData := `{"username": "John", "age": 30}`
var m map[string]interface{}

dec := json.NewDecoder(strings.NewReader(myData))
dec.UseNumber()

err := dec.Decode(&m)
if err != nil {
     log.Fatal(err)
}

m["age"], err = m["age"].(json.Number).Int64()
if err != nil {
	log.Fatal(err)
}
```

Reply:
Ok totally agree

Reply:
Is this thing happening on other SDKs as well or only in Go? if no so let's add a comment about it in the readme file of Go sdk it is enough